### PR TITLE
coodesker.Coodesker version 1.0.4.1

### PR DESCRIPTION
**Pull request by Lograthmic**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/107686)

### CODE REVIEW

**Comment by Lograthmic** [comment]
> 

@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

File: manifests/c/coodesker/Coodesker/1.0.4.1/coodesker.Coodesker.locale.zh-CN.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: coodesker.Coodesker
+PackageVersion: 1.0.4.1
+PackageLocale: zh-CN
+Publisher: coodesker
+PackageName: Coodesker
+License: Unknow
```
Comment (stephengillie) [inline manifests/c/coodesker/Coodesker/1.0.4.1/coodesker.Coodesker.locale.zh-CN.yaml:9]:
```suggestion
License: Proprietary
```

File: manifests/c/coodesker/Coodesker/1.0.4.1/coodesker.Coodesker.locale.zh-CN.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: coodesker.Coodesker
+PackageVersion: 1.0.4.1
+PackageLocale: zh-CN
+Publisher: coodesker
+PackageName: Coodesker
+License: Unknow
+Copyright: Copyright (C) 2023
+ShortDescription: Coodesker
```
Comment (Trenly) [inline manifests/c/coodesker/Coodesker/1.0.4.1/coodesker.Coodesker.locale.zh-CN.yaml:11]: Please provide a better short description of what this program does

File: manifests/c/coodesker/Coodesker/1.0.4.1/coodesker.Coodesker.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: coodesker.Coodesker
+PackageVersion: 1.0.4.1
+Installers:
+- Architecture: x64
```
Comment (Trenly) [inline manifests/c/coodesker/Coodesker/1.0.4.1/coodesker.Coodesker.installer.yaml:7]: Please also add the x86 version - https://github.com/coodesker/coodesker-desktop/releases/download/v1.0.4.1/Coodesker-x86_1.0.4.1.exe

**Review by Trenly — CHANGES_REQUESTED** [review]

**Pull request closed without merging**